### ISSUE TITLE
conda env create error

### ISSUE THREAD

**Issue by w4n9H** [byzer-org/byzer-lang#915]
SQL语句为
```sql
load csv.`wine-quality.csv`  options header="true"  and inferSchema="true"  as t1;

train t1 as PythonAlg.`mltest` 
where pythonScriptPath="MLtest" 
and keepVersion="true" 
and dataLocalFormat="csv"
and `systemParam.envs`='''{"MLFLOW_CONDA_HOME": "/opt/miniconda3"}'''
as res;

save overwrite res as json.`new_pyml_result`;
```
使用spark-submit通过yarn向集群提交任务，出现报错  java.lang.RuntimeException: Nonzero exit value: 1，错误出现在Project.scala中创建conda env的那一步
打印详细的create语句，出现了多条
/opt/miniconda3/bin/conda env create -n mlflow-a6e270601633eae621911dbfd740bdebe63e458a --file /tmp/27af026b-f309-499f-a536-df14ceb42a20.yaml
还有一条为
/opt/miniconda3/bin/conda env create -n mlflow-a6e270601633eae621911dbfd740bdebe63e458a --file /tmp/4bf24767-853f-41ac-9575-79925eb03c85.yaml
看这个报错应该是 --file 后面的文件不存在导致的
使用local模式在本地跑是正常的

**Comment by allwefantasy** [comment]
Have you configured your cluster by this [tutorial](http://docs.mlsql.tech/v1.1.7/zh/installation/cluster-mode.html)?

**Comment by allwefantasy** [comment]
Also there are some bugs about python in v1.1.7 . more detail please check [here](https://github.com/allwefantasy/streamingpro/issues/905), and we will fix it  in bug-release version  v.1.1.7.1 and  maybe this weekend.

**Comment by allwefantasy** [comment]
We have released [v1.1.7.1](https://www.jianshu.com/p/c0d3f99b1e82) to fix this issue. 
The doc is [here- how to manage you python environment](http://docs.mlsql.tech/v1.1.7.1/zh/python/python-env.html)

We will close this issue. If you find any problem again ,please reopen it or create a new one .
